Makefile

```diff
@@ -2,12 +2,11 @@ CC = gcc
 AR = ar 
 CFLAGS = -std=c99 -fPIC -Wall -Wextra -O2 -g -I./include/
 LDFLAGS = -shared
-RM = rm -f
+RM = rm -rf
```

Comment:
What is this flag (-rf)?

Reply:
Specifically, it allows the deletion of *.d and *.o files from folder that aren't the main folder, like /srcs.